alerts/tplink.markdown

```diff
@@ -0,0 +1,9 @@
+---
+title: "TP-Link Remove Local API"
+created: 2020-11-18 09:08:00
+integrations:
+  - tplink
+homeassistant: ">0.89"
+---
+
+In response to a security vulnerability TP-Link have removed their local management API for Kasa products in the latest firmware version which prevents Home Assistant communicating with the device. Please see [this tweet](https://twitter.com/TPLINKUK/status/1328687659133399043) for details from TP-Link, release notes for firmware are not readily published by TP-Link. Please see [this discussion on our community forums](https://community.home-assistant.io/t/tp-link-hs110-smart-plug-disappears-after-latest-firmware-update/244229) for more details.
```

Comment:
"security vulnerability" has not been confirmed. It's the reason that TP-Link has given.

Reply:
Rephrased, luckily it sounds like there may an alternative port available. Seem's we're using an "abandoned" library anyway ([PyHS100](https://github.com/GadgetReactor/pyHS100)) should probably switch to [Python-Kasa](https://github.com/python-kasa/python-kasa) when the fix gets merged.